test/integ/test_cursor.py

```diff
@@ -929,7 +935,9 @@ def test_rownumber(conn):
     """Checks whether rownumber is returned as expected."""
     with conn() as cnx:
         with cnx.cursor() as cur:
-            assert cur.execute("select * from values (1), (2)").fetchone() == (1,)
+            assert cur.execute("select * from values (1), (2)")
+            assert cur.rownumber is None
```

Comment:
we want to test if the rownumber is None before fetchone?

Reply:
Why not? It's the expected behavior as per https://www.python.org/dev/peps/pep-0249/#rownumber

Reply:
Nice!